extensions/adobe/experience/decisioning/eventmetrics.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/eventmetrics",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision reporting event metrics",
+  "type": "object",
+  "description": "For decisions events and user reactions, we capture certain metrics along with the dimensions. This schema hold the metrics associated with a decisioning activity",
+  "definitions": {
+    "decisioingeventmetrics-datatype": {
+      "properties": {
+        "https://ns.adobe.com/experience/offer-management/offer-propositions": {
+          "title": "Total number of time a particular offer was proposed",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Total number of time a particular offer was proposed"
```

Comment:
Total *amount* of time

Also end descriptions with a period "."

Reply:
Fixed